index.js

```diff
@@ -23,16 +23,23 @@ function getHash(data) {
  * @param {string} src source of the file being compiled to ES5
  * @return {string} compiled source
  */
-function compileFile(file, src) {
+function compileFile(file, src, opts) {
   var compiled;
-  compiled = compile(file, src, exports.traceurOverrides);
+  opts.traceurOverrides = exports.traceurOverrides;
+  compiled = compile(file, src, opts);
   if (compiled.error) throw new Error(compiled.error);
 
   return compiled.source;
 }
 
-function es6ify(filePattern) {
-  filePattern =  filePattern || /\.js$/;
+function es6ify(opts) {
+  opts || (opts = {});
+
+  // This function's argument used to be filePattern (regex), so duck type opts
+  // to see if it's meant as filePattern.
+  if (typeof opts.ignoreCase !== 'undefined') opts = {filePattern: opts};
```

Comment:
I'll proceed on that premise unless I hear otherwise. Since that will make it more verbose to specify `filePattern`, do you want to take this opportunity to shorten that (`patt` maybe)?